client/src/types/game/game.types.tsx

```diff
@@ -4,5 +4,5 @@ export type Game = {
   players: Player[];
   totalRounds: number;
   currentRound: number;
-  roundWinners: Player[];
+  roundWinners: string[];
```

Comment:
Changed as agreed with @beccarothwell who's done the same in her PR on implementing the round.